### PR TITLE
AIC Poisson/Neg-Bin Implementation

### PR DESCRIPTION
**Pull request by ryan-odea**
Added
- Formula_List, develops list of possible formulas given strings
- AIC methodology for Poisson and Negative Binomial Regression off of the brute force formula_list creation
- At convergence failure the `crc` function will report the error and then continue on with the rest of the brute force methods

Fixed
- Removed named numerics output from `confint()` = looks nicer, a little nicer to work with on the user side
- Removed `confint` spam into the console
- Fixed a spelling error :)

### CODE REVIEW

**Review by ddbaptiste — APPROVED** [review]

**Review by MJC598 — APPROVED** [review]

File: R/crc.R [hunk 1]
```diff
@@ -38,34 +34,128 @@ crc <- function(data, freq.column, binary.variables, method = "poisson", corr.th
   corr_plot <- ggcorrplot::ggcorrplot(corr, type = "upper", lab = TRUE) +
     ggplot2::theme(legend.position = "none")
 
-  #Poisson Modeling ===================================
+  if(is.null(formula)){
```
Comment (MJC598) [inline R/crc.R:37]: This checks to verify formula is null and then the nested if statements uses formula, wouldn't the nested if statements require it to be is.not.null(formula) to work?

Comment (MJC598) [inline R/crc.R:37]: This checks to verify formula is null and then the nested if statements uses formula, wouldn't the nested if statements require it to be is.not.null(formula) to work?

**Review by MJC598 — CHANGES_REQUESTED** [review]
Just the one comment, but it seems like it could force no formula on the data

**Review by MJC598 — APPROVED** [review]

**Review by MJC598 — COMMENTED** [review]
Cool

**Pull request merged**